package.json

```diff
@@ -34,9 +34,9 @@
     "generate:abis": "cd ./abis && ./_generate-abis.sh && cd ../"
   },
   "dependencies": {
-    "@artblocks/contracts": "1.0.3",
+    "@artblocks/contracts": "1.0.4",
     "@assemblyscript/loader": "^0.19.22",
-    "@graphprotocol/graph-cli": "^0.37.1",
+    "@graphprotocol/graph-cli": "^0.56.0",
```

Comment:
had to do this due to an old dependency in old version randomly breaking, which broke e2e testing